classes/class-list-table.php

```diff
@@ -545,6 +549,31 @@ public function get_filters() {
 		return apply_filters( 'wp_stream_list_table_filters', $filters );
 	}
 
+	function export_link() {
+
+		$export_links = array();
+		$exporters = apply_filters( 'stream_exporters', array() );
+		if ( empty( $exporters ) ) {
+			return;
+		}
+		
+		foreach ( array_keys( $exporters ) as $export_type ) {
+			$args = array_merge( array( 'output' => $export_type ), $_GET );
+			$download = add_query_arg( $args, 'admin.php' );
+
+			$export_links[] = sprintf(
+				'<a href="%s">%s</a>',
+				esc_html( $download ),
+				esc_html( strtoupper ( $export_type ) )
```

Comment:
See comment above. This will change to something like `esc_html( $this->admin->export->exporters[ $export_type ]->name )`.